### ISSUE TITLE
"To many errors. (XX% scanned)" warning in Postman Sandbox

### ISSUE THREAD

**Issue by serkand** [a85/POSTMan-Chrome-Extension#3385]
I am getting this "To many errors. (XX% scanned)" warning in some of my pre-req and test scripts. 

When I copy the code to a jslint validator ( say in Visual Studio Code or an online validator) I can see the errors. Even though most are warnings and quite trivial which can be ignored, they are annoying!!!

Here are some examples: 
Postman's data object:
file: 'file:///c%3A/Users/.../Desktop/temp.js'
severity: 'Warning'
message: 'Undeclared 'data'. (undeclared_a)'
at: '1,16'
source: 'jslint'

The JS console.log function:
file: 'file:///c%3A/Users/.../Desktop/temp.js'
severity: 'Warning'
message: 'Undeclared 'console'. (undeclared_a)'
at: '34,17'
source: 'jslint'

My question is, is there a way to change the **JSLint** warning tolerance level in the Postman Sandbox?
Or be able to update a setting (somewhere???) to ignore such warnings?

## App Details:
Postman for Windows
Version 5.1.3
win32 10.0.15063 / x64

**Comment by kamalaknn** [comment]
Hey @serkand 

I don't think the Postman Sandbox is warning you on lint errors. That could have been a call stack overload or a memory overload. Could you share a sample script, a minimal version of it, where you are seeing this? So we can reproduce this. Thanks.

**Comment by serkand** [comment]
Here is a screen shot from Postman:

![image](https://user-images.githubusercontent.com/22509908/29348249-8cab0b66-8295-11e7-910f-3483021ead0c.png)

**Comment by serkand** [comment]
Let me know if you require the script in a text format.

**Comment by kamalaknn** [comment]
@serkand the text format would really help to look at the code once.

**Comment by serkand** [comment]
@kamalaknn i've emailed you the text version

**Comment by serkand** [comment]
@kamalaknn haven't heard anything so just wondering if you actually received my email ?

**Comment by kamalaknn** [comment]
@serkand I've received your email. I haven't had the time to dive deep yet. Will update this thread once I've reproduced this.

**Comment by kamalaknn** [comment]
@serkand This is coming the script editor trying to parse your script. There would be no runtime errors while using this though. 

We've updated our script editor in the recent releases and I see that there are no warnings now. 

I'm closing this issue now. Thanks

**Comment by BhanuDash007** [comment]
Hi, I am getting this error . If you write 100 lines of code this issue is coming . I checked the syntax other editor no issue is thr

**Comment by numaanashraf** [comment]
@BhanuDash007 Could you share your app version and a sample script?

**Comment by BhanuDash007** [comment]
Version 6.3.0.

**Comment by BhanuDash007** [comment]
pm.test("TC_14 : Test ", function () {
   
    var jsonData=pm.response.json();
    var Postman_Webservice=jsonData.postmanlab;
    
   for(i=0; i<Postman_Webservice.length; i+=1){
       pm.expect(Postman_Webservice[i].char).to.eql("");
       pm.expect(Postman_Webservice[i].delS).to.eql("");
	   pm.expect(Postman_Webservice[i].spcl).to.eql("");
       pm.expect(Postman_Webservice[i].money).to.eql("");
       pm.expect(Postman_Webservice[i].service).to.eql("");
       pm.expect(Postman_Webservice[i].distribution).to.eql("");
	   pm.expect(Postman_Webservice[i].stdDays).to.eql("");
       pm.expect(Postman_Webservice[i].stdTime).to.eql("");
    }
});

**Comment by BhanuDash007** [comment]
Any update?

**Comment by numaanashraf** [comment]
@BhanuDash007 We are unable to replicate this issue. Could you share a screenshot and detailed steps for replicating this issue?

<img width="849" alt="screen shot 2018-09-21 at 9 18 33 am" src="https://user-images.githubusercontent.com/3295568/45859347-6a7eef00-bd7f-11e8-93ec-ac7ca960b1cf.png">

**Comment by GregoryPellegrin** [comment]
I have the same warning with this in Tests Tab


```
pm.test("Le code de retour est valide", function () {
    pm.response.to.have.status(200);
});

pm.test("Le format de la réponse est valide", function () {
    var jsonData = pm.response.json();

    pm.expect(typeof jsonData === 'object').to.eql(true);
    pm.expect(typeof jsonData.produitEntente === 'object').to.eql(true);
    pm.expect(typeof jsonData.typeEntenteLiee === 'object').to.eql(true);
    pm.expect(typeof jsonData.statuts === 'object').to.eql(true);
    pm.expect(typeof jsonData.statuts.statutCourant === 'object').to.eql(true);
    pm.expect(typeof jsonData.statuts.statutCourant.statutSimplifie === 'object').to.eql(true);
    pm.expect(typeof jsonData.statuts.statutCourant.statutDetaille === 'object').to.eql(true);
    pm.expect(typeof jsonData.infoCreditVariable === 'object').to.eql(true);
    pm.expect(typeof jsonData.infoFinancement === 'object').to.eql(true);
});

pm.test("Les champs demandés sont présents", function () {
    pm.expect(pm.response.text()).to.include("localisationEntente");
    pm.expect(pm.response.text()).to.include("adresseListe");
    pm.expect(pm.response.text()).to.include("adresseLigne1");
    pm.expect(pm.response.text()).to.include("ville");
    pm.expect(pm.response.text()).to.include("codeProvince");
    pm.expect(pm.response.text()).to.include("codePostal");
    pm.expect(pm.response.text()).to.include("dateEffective");
    pm.expect(pm.response.text()).to.include("indicateurBonneAdresse");
    pm.expect(pm.response.text()).to.include("codeCorrespondancePostale");
    pm.expect(pm.response.text()).to.include("courriel");
    pm.expect(pm.response.text()).to.include("produitEntente");
    pm.expect(pm.response.text()).to.include("fiid");
    pm.expect(pm.response.text()).to.include("typeCompte");
    pm.expect(pm.response.text()).to.include("codeLangue");
    pm.expect(pm.response.text()).to.include("institutionFinanciere");
    pm.expect(pm.response.text()).to.include("noInstitution");
    pm.expect(pm.response.text()).to.include("noTransit");
    pm.expect(pm.response.text()).to.include("typeEntenteLiee");
    pm.expect(pm.response.text()).to.include("dateOuverture");
    pm.expect(pm.response.text()).to.include("dateExpiration");
    pm.expect(pm.response.text()).to.include("dateExpirationPrecedente");
    pm.expect(pm.response.text()).to.include("fermeture");
    pm.expect(pm.response.text()).to.include("raisonFermeture");
    pm.expect(pm.response.text()).to.include("detailFermeture");
    pm.expect(pm.response.text()).to.include("numeroCompte");
    pm.expect(pm.response.text()).to.include("statuts");
    pm.expect(pm.response.text()).to.include("dateMAJStatut");
    pm.expect(pm.response.text()).to.include("statutCourant");
    pm.expect(pm.response.text()).to.include("statutSimplifie");
    pm.expect(pm.response.text()).to.include("statutDetaille");
    pm.expect(pm.response.text()).to.include("statutPrecedent");
    pm.expect(pm.response.text()).to.include("infoCreditVariable");
    pm.expect(pm.response.text()).to.include("montantLimite");
    pm.expect(pm.response.text()).to.include("montantSolde");
    pm.expect(pm.response.text()).to.include("montantSoldeTotal");
    pm.expect(pm.response.text()).to.include("montantTotalAutoriseesNonFacturees");
    pm.expect(pm.response.text()).to.include("montantCreditDisponible");
    pm.expect(pm.response.text()).to.include("dateDerniereMAJLimite");
    pm.expect(pm.response.text()).to.include("montantLimitePrecalculee");
    pm.expect(pm.response.text()).to.include("montantLimiteSuggereeTRIAD");
    pm.expect(pm.response.text()).to.include("montantTotalGelSIC");
    pm.expect(pm.response.text()).to.include("montantTotalReservationSIC");
    pm.expect(pm.response.text()).to.include("montantAutoriseAugmente");
    pm.expect(pm.response.text()).to.include("montantRetenuePaiement");
    pm.expect(pm.response.text()).to.include("codePaiementPreautorise");
    pm.expect(pm.response.text()).to.include("montantPaiementFixe");
    pm.expect(pm.response.text()).to.include("montantSoldeAvancesFonds");
    pm.expect(pm.response.text()).to.include("limiteExtension");
    pm.expect(pm.response.text()).to.include("interets");
    pm.expect(pm.response.text()).to.include("montantInteretAchat");
    pm.expect(pm.response.text()).to.include("montantInteretAVF");
    pm.expect(pm.response.text()).to.include("montantInteretPetrol");
    pm.expect(pm.response.text()).to.include("montantInteretCrediteur");
    pm.expect(pm.response.text()).to.include("montantTotalAchats");
    pm.expect(pm.response.text()).to.include("montantTotalPaiements");
    pm.expect(pm.response.text()).to.include("montantTotalAvancesFonds");
    pm.expect(pm.response.text()).to.include("montantTotalRedressements");
    pm.expect(pm.response.text()).to.include("infoFinancement");
    pm.expect(pm.response.text()).to.include("comportementCredit");
    pm.expect(pm.response.text()).to.include("indicateurLimiteMax");
    pm.expect(pm.response.text()).to.include("raisonLimiteMax");
    pm.expect(pm.response.text()).to.include("montantDepassementLimite");
    pm.expect(pm.response.text()).to.include("montantSouffrance");
    pm.expect(pm.response.text()).to.include("montantSouffranceMaximal");
    pm.expect(pm.response.text()).to.include("indiceBNI");
    pm.expect(pm.response.text()).to.include("indiceBHV");
    pm.expect(pm.response.text()).to.include("indiceBHVCourant");
    pm.expect(pm.response.text()).to.include("indiceBHVHistoriqueListe");
    pm.expect(pm.response.text()).to.include("indicateurAvanceFond");
    pm.expect(pm.response.text()).to.include("comportementPaiement");
    pm.expect(pm.response.text()).to.include("codePaiement");
    pm.expect(pm.response.text()).to.include("codePaiementActuel");
    pm.expect(pm.response.text()).to.include("codeUtilisationLimite");
    pm.expect(pm.response.text()).to.include("cycleRetard");
    pm.expect(pm.response.text()).to.include("cycleRetardHistoriqueParMoisListe");
    pm.expect(pm.response.text()).to.include("cycleRetardHistoriqueParNbCycleListe");
    pm.expect(pm.response.text()).to.include("montantPaiementMinimumRestant");
    pm.expect(pm.response.text()).to.include("indicateurCodetention");
    pm.expect(pm.response.text()).to.include("cycleFacturation");
});

pm.test("La recherche a trouvé les bons résultats", function () {
    var jsonData = pm.response.json();

    pm.expect(jsonData.numeroCompte).to.eql(pm.environment.get("compteNo-Existant"));
});
```

**Comment by GregoryPellegrin** [comment]
![image](https://user-images.githubusercontent.com/2473943/47111745-5addb380-d222-11e8-9861-d721ab9585eb.png)

**Comment by tomkidron** [comment]
I've encountered the same issue, using "Postman for windows" Version 6.7.2.
After trying every js linter known to man, i think i found a possible solution\workaround:

1. Download an IDE (i used VS Code)
2. Download and enable the jslint extension (install jslint using npm, if needed)
3. Copy and paste the test script from postman to VS Code
4. Fix all errors and warning detected by the jslint extension
5. Copy\paste the fixed code back to postman

Voila, no errors.

**Comment by tretex** [comment]
Any updates on the issue? I think this happens even if you don't have any particular problems in your code. Sometimes exclamation mark icon is shown but without "Too many errors" note. 

This is particularly annoying problem when using common code for testing multiple requests.

**Comment by ubz8a** [comment]
My problem is  a little simpler but in same vein.

If you have 100 lines of code in Pre-requst  script or Tests UI then the next line is flagged has having too many errors
where a line of code is delimited by a semi-colon , so 101 semi-colons on a single line will return error

e.g. NO error ...  100  * a=1;

a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;a=1;

Error 101 * a=1;

Add **a=;** to the line aboveand you wil get the error

**Comment by pankajk07** [comment]
Fixed as part of v7.24
